# The code will output an error as the function call on the final 
# line does not give enough arguments 

def foo(bar, qux):
    print(bar)
    print(qux)

foo('Hello')